build_library/catalyst.sh

```diff
@@ -231,7 +231,7 @@ write_configs() {
     info "Configuring Portage..."
     cp -r "${BUILD_LIBRARY_DIR}"/portage/ "${TEMPDIR}/"
 
-    ln -sfT '/mnt/host/source/src/third_party/coreos-overlay/coreos/user-patches' \
+    ln -sfT '/var/gentoo/repos/flatcar-overlay/coreos/user-patches' \
```

Comment:
same question about `/var/gentoo/repos` as above - can we stick to the same paths as used elsewhere?

Reply:
Fixed.